java-jacoco/src/main/java/org/sonar/plugins/jacoco/JacocoConstants.java

```diff
@@ -0,0 +1,103 @@
+    String subCategory = "JaCoCo";
+    if (sqVersion.isGreaterThanOrEqual(SQ_6_2)) {
+      return ImmutableList.of(
+        PropertyDefinition.builder(JacocoConstants.REPORT_PATHS_PROPERTY)
+          .defaultValue(JacocoConstants.REPORT_PATHS_DEFAULT_VALUE)
+          .category(CoreProperties.CATEGORY_JAVA)
+          .subCategory(subCategory)
+          .name("JaCoCo Reports")
+          .description("Path to the JaCoCo report files containing coverage data. The paths may be absolute or relative to the project base directory.")
+          .onQualifiers(Qualifiers.PROJECT, Qualifiers.MODULE)
+          .build(),
+        PropertyDefinition.builder(JacocoConstants.REPORT_PATH_PROPERTY)
+          .defaultValue(JacocoConstants.REPORT_PATH_DEFAULT_VALUE)
+          .category(CoreProperties.CATEGORY_JAVA)
+          .subCategory(subCategory)
+          .name("UT JaCoCo Report [DEPRECATED]")
+          .description("Deprecated since SonarQube 6.2. Use '" + JacocoConstants.REPORT_PATHS_PROPERTY + "' instead.")
+          .onQualifiers(Qualifiers.PROJECT, Qualifiers.MODULE)
+          .build(),
+        PropertyDefinition.builder(JacocoConstants.IT_REPORT_PATH_PROPERTY)
+          .defaultValue(JacocoConstants.IT_REPORT_PATH_DEFAULT_VALUE)
+          .category(CoreProperties.CATEGORY_JAVA)
+          .subCategory(subCategory)
+          .name("IT JaCoCo Report [DEPRECATED]")
+          .description("Deprecated since SonarQube 6.2. Use '" + JacocoConstants.REPORT_PATHS_PROPERTY + "' instead.")
+          .onQualifiers(Qualifiers.PROJECT, Qualifiers.MODULE)
+          .build());
+    } else {
+      return ImmutableList.of(
+        PropertyDefinition.builder(JacocoConstants.REPORT_PATH_PROPERTY)
+          .defaultValue(JacocoConstants.REPORT_PATH_DEFAULT_VALUE)
+          .category(CoreProperties.CATEGORY_JAVA)
+          .subCategory(subCategory)
+          .name("UT JaCoCo Report")
+          .description("Path to the JaCoCo report file containing coverage data by unit tests. The path may be absolute or relative to the project base directory.")
+          .onQualifiers(Qualifiers.PROJECT, Qualifiers.MODULE)
+          .build(),
+        PropertyDefinition.builder(JacocoConstants.IT_REPORT_PATH_PROPERTY)
+          .defaultValue(JacocoConstants.IT_REPORT_PATH_DEFAULT_VALUE)
+          .category(CoreProperties.CATEGORY_JAVA)
+          .subCategory(subCategory)
+          .name("IT JaCoCo Report")
+          .description("Path to the JaCoCo report file containing coverage data by integration tests. The path may be absolute or relative to the project base directory.")
+          .onQualifiers(Qualifiers.PROJECT, Qualifiers.MODULE)
+          .build(),
+        PropertyDefinition.builder(JacocoConstants.REPORT_MISSING_FORCE_ZERO)
+          .defaultValue(Boolean.toString(JacocoConstants.REPORT_MISSING_FORCE_ZERO_DEFAULT_VALUE))
+          .name("Force zero coverage")
+          .category(CoreProperties.CATEGORY_JAVA)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "CATEGORY_JAVA"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)